src/components/targets/CreateTargetForm.js

```diff
@@ -75,13 +75,13 @@ const CreateTargetForm = ({
 
   const handleChangeTopic = (select_topic) => {
     setInputs((inputs) => ({ ...inputs, topic: select_topic['value'] }));
-    setErrors(omit(errors, 'topic'));
+    setErrors(omit(errors, targetFormNames.topic));
```

Comment:
you could destructure the object at the beginning of the component, so you avoid _targetFormNames_ in all the sentences.
```suggestion
    const { topic, radius, title } = targetFormNames;
    ...
    setErrors(omit(errors, topic));
```

Reply:
Great idea